test/cqlpy/test_describe.py

```diff
@@ -1595,7 +1595,7 @@ def __init__(self, cql, ks=None):
 
     def __enter__(self):
         if self.ks:
-            self.cql.execute(f"CREATE KEYSPACE {self.ks} WITH REPLICATION = {{ 'class': 'SimpleStrategy', 'replication_factor': 1 }}")
+            self.cql.execute(f"CREATE KEYSPACE {self.ks} WITH REPLICATION = {{ 'class': 'NetworkTopologyStrategy', 'replication_factor': 1 }}")
```

Comment:
Maybe this is a good change, but it's even less clear how it relates to vector search and probably doesn't belong in this patch?